### DISCUSSION TITLE
Use PixArt-sigma to train my own 512*512 dataset

### DISCUSSION THREAD

**Discussion by zmy-08** [PixArt-alpha/PixArt-alpha#129] [Q&A]
Very lucky to use PixArt-sigma! I have used Pixart-alpha and Pixart-δ before, and now I want to use PixArt-sigma to train my own 512*512 data set. **Currently I have a V100 with 32G. I want to know if I can train PixArt-sigma?** **Is 3090 okay?** **Approximately how long?** **Compared with Pixart-lcm and PixArt-sigma, which one has lower training requirements?** Sorry, I'm a college freshman, so I'm interrupting your time.

**Comment by lawrence-cj** [comment]
They are using very similar training cost. Feel free to train with PixArt-Sigma
